packages/core/src/utils/renderThumbs.ts

```diff
@@ -0,0 +1,45 @@
+export function renderThumbs(
+  current: number,
+  total: number,
+  urls: Array<string>,
+  videoIcons: boolean
+): string {
+  const thumbs = []
+
+  if (!urls.length) return ''
```

Comment:
(question) `url` could be empty?
```suggestion
  if (!urls?.length) return ''
```

Reply:
in this case no, because in the carousel file it has a verify in case that the string is empty, and the parameter is required and well typed, the only case is that someone pass null as param in code

Reply:
that line always return an array
`const urls = !this.thumbnailImages ? [] : this.thumbnailImages.split(',')`

Reply:
> the only case is that someone pass null as param in code

Don't you think it's safer to maintain optional chaining to avoid any such scenario?